openmrs/src/main/java/org/motechproject/openmrs/service/OpenMRSPatientService.java

```diff
@@ -75,6 +75,17 @@
     Patient getPatientByMotechId(String configName, String motechId);
 
     /**
+     * Returns the patient with the given {@code otherName} and {@code otherId}. Configuration with the given {@code configName} will be used
```

Comment:
we should not use the word 'other', it's not obvious what it means. Just name it 'identifier' without other